test/python/overland_tiltedV_MGSemiTest.py

```diff
@@ -0,0 +1,511 @@
+
+overland.KnownSolution = 'NoKnownSolution'
+
+#-----------------------------------------------------------------------------
+# Set solver parameters
+#-----------------------------------------------------------------------------
+
+overland.Solver = 'Richards'
+overland.Solver.MaxIter = 2500
+
+overland.Solver.Nonlinear.MaxIter = 100
+overland.Solver.Nonlinear.ResidualTol = 1e-9
+overland.Solver.Nonlinear.EtaChoice = 'EtaConstant'
+overland.Solver.Nonlinear.EtaValue = 0.01
+overland.Solver.Nonlinear.UseJacobian = False
+overland.Solver.Nonlinear.DerivativeEpsilon = 1e-15
+overland.Solver.Nonlinear.StepTol = 1e-20
+overland.Solver.Nonlinear.Globalization = 'LineSearch'
+overland.Solver.Linear.KrylovDimension = 50
+overland.Solver.Linear.MaxRestart = 2
+overland.Solver.OverlandKinematic.Epsilon = 1E-5
+
+overland.Solver.Linear.Preconditioner = 'MGSemi'
+overland.Solver.PrintSubsurf = False
+overland.Solver.Drop = 1E-20
+overland.Solver.AbsTol = 1E-10
+
+overland.Solver.WriteSiloSubsurfData = False
+overland.Solver.WriteSiloPressure = False
+overland.Solver.WriteSiloSlopes = False
+
+overland.Solver.WriteSiloSaturation = False
+overland.Solver.WriteSiloConcentration = False
+
+#---------------------------------------------------------
+# Initial conditions: water pressure
+#---------------------------------------------------------
+
+# set water table to be at the bottom of the domain, the top layer is initially dry
+overland.ICPressure.Type = 'HydroStaticPatch'
+overland.ICPressure.GeomNames = 'domain'
+overland.Geom.domain.ICPressure.Value = -3.0
+
+overland.Geom.domain.ICPressure.RefGeom = 'domain'
+overland.Geom.domain.ICPressure.RefPatch = 'z_upper'
+
+
+
+#-----------------------------------------------------------------------------
+# New kinematic formulations without the zero channel
+# Note: The difference in configuration here is to be consistent with the way
+# the upwinding is handled for the new and original formulations.
+# These two results should be almost identical for the new and old formulations
+#-----------------------------------------------------------------------------
+overland.TopoSlopesX.Type = 'Constant'
+overland.TopoSlopesX.GeomNames = 'left right channel'
+overland.TopoSlopesX.Geom.left.Value = -0.01
+overland.TopoSlopesX.Geom.right.Value = 0.01
+overland.TopoSlopesX.Geom.channel.Value = 0.01
+
+overland.TopoSlopesY.Type = 'Constant'
+overland.TopoSlopesY.GeomNames = 'domain'
+overland.TopoSlopesY.Geom.domain.Value = 0.01
+
+# run with KWE upwinding
+overland.Patch.z_upper.BCPressure.Type = 'OverlandKinematic'
+overland.Solver.Nonlinear.UseJacobian = False
+#overland.Solver.Linear.Preconditioner.PCMatrixType = 'PFSymmetric'
+
+
+run_name = "TiltedV_OverlandKin_JacFalse_MGSemi"
+overland.set_name(run_name)
+print("##########")
+print(f"Running {run_name}")
+new_output_dir_name = get_absolute_path('test_output/' + f"{run_name}")
+mkdir(new_output_dir_name)
+overland.run(working_directory=new_output_dir_name)
```

Comment:
Need to check output of the run.   Simply running doesn't indicate very much.   No idea what the result converged to.